cli/utilities/containers.py

```diff
@@ -54,25 +54,27 @@ def __init__(self, nodes, package="podman"):
     def run(
         self,
         image=None,
-        name=None,
         rm=None,
+        name=None,
         env=None,
         volume=None,
         ports=None,
         restart=None,
-        detach=None,
+        detach_key=None,
+        detach=False,
         cmds=None,
     ):
         """Executes the provided command using podman
         Args
             image (str): Image name
+            rm (str): Image name to remove from background
             name (str): Container name
             env (list): List of environment variables
             volume (list): List of volumes
             ports (list): List of ports
-            rm (str): Image name to remove from background
-            detach (str): Image name to detach
             restart (str): Restart flag
+            detach_key (list): List of detach operation
+            detach (str): Image name to detach
             cmds (str): Other commands to be executed
         """
         if not image or not rm:
```

Comment:
```suggestion
        if not image and not rm:
```
The condition will not be fulfilled when passing only the image.

Reply:
Fixed now